wordpress-mobile/WordPress-iOS
# reverting project file

Fixes #9286 

To test:
See the CI passes
This is a PR to revert the project file to fix Ci failure to find a file:
error: Build input file cannot be found: '/Users/distiller/project/WordPress/WordPressDraftActionExtension/zh-Hant.lproj/InfoPlist.strings'
Update release notes:
